source/configuring_domain/05_configuring_domain/tabs.rst

```diff
@@ -20,7 +20,7 @@ Dataset page
 ------------
 
 Users can explore and analyze datasets using different visualizations and tools, each one available in a tab in the dataset page.
-See :doc:`Visualizing data </exploring_catalog_and_datasets/02_visualizing_data/dataset>` for more details.
+See :doc:`Visualizing data </exploring_catalog_and_datasets/02_looking_up_a_dataset/visualizing_data>` for more details.
 
 **Default tab**
```

Comment:
For the content below this title

> The table view by default is useful to give users with a direct access to the data, whereas the information tab by default is interesting to provide users with context and description first.

The table view by default is useful to give users ~~with~~ a direct access to the data, whereas the information tab by default is **an alternative** to provide users with context and description first.